src/runtime_src/core/tools/common/Escape_Codes.h

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (C) 2020 Xilinx, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef __ESCAPE_CHAR_H__
+#define __ESCAPE_CHAR_H__
+
+#include <iostream>
+#include <string>
+
+namespace ec {
```

Comment:
spell it out  would be better..  It's nice when namespace (or class) matches the name of the file.